internal/agent/agent.go

```diff
@@ -375,6 +375,7 @@ func (a *DeviceAgent) ensureManagementClient() error {
 				TLSClientConfig: &tls.Config{
 					RootCAs:      a.caCertPool,
 					Certificates: []tls.Certificate{*a.managementClientCert},
+					MinVersion:   tls.VersionTLS12,
```

Comment:
The server uses min TLS 1.3: https://github.com/flightctl/flightctl/blob/49a37ef878677399c7aa6ff7475bddf08156c968/internal/crypto/tls.go#L32

Reply:
Fixed